s/nub-context/nub-context.ts

```diff
@@ -0,0 +1,70 @@
+import {dispatchCustomEvent} from "../nub-keyboard/utils/dispatchCustomEvent.js"
+import {keys} from "../nub-keyboard/utils/keys.js"
+import {styles} from "./style.css.js"
+
+export class NubContext extends HTMLElement {
+	shadow = this.attachShadow({
+		mode: "open",
+		delegatesFocus: false,
+	})
+
+	constructor() {
+		super()
+		let editable = false
+		this.shadow.innerHTML = `
+		<div>
+		<button class="flip">flip mode</button>
+		<slot>
+		</slot>
+		</div>
+		`
+		const style = document.createElement("style")
+		style.textContent = styles
+		this.shadowRoot?.append(style)
+		const flipButton = this.shadowRoot?.querySelector<HTMLElement>(".flip")
+		const nubContext = document.querySelector("nub-context")
```

Comment:
something strange is happening on this line:

```ts
const nubContext = document.querySelector("nub-context")
```

we should not need to query for the nub-context element -- we *are* the nub-context element -- we can refer simply to the nub-context element as `this`